Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -287,6 +287,29 @@ bool GameListItem::ReadPNGBanner(const std::string& path)
   return true;
 }
 
+const std::string GameListItem::GetFileType() const
+{
+  switch (GetBlobType())
+  {
+  case DiscIO::BlobType::PLAIN:
+    return wxTRANSLATE("Plain");
```

Comment:
I orignally planned to to the actual translation in the `GameListCtrl.cpp`, but forgot it. I did it like that because no other method in `ISOFile` return a wxString, so I wanted to be consistent. So would it be fine if I put use `_()` in `GameListCtrl.cpp`?